### ISSUE TITLE
Deleting a release from Skipper shell breaks the stream in Data Flow

### ISSUE THREAD

**Issue by emedina** [spring-cloud/spring-cloud-dataflow#3492]
**Description:**
I create a simple stream (file -> filter -> log) in Data Flow UI. Then I deploy it. Then I run the Skipper shell and `release list` shows me the current deploying of my stream. But if I use Skipper's shell to delete the release, i.e. cancel the deployment of the stream, then apparently it works (the stream's deployment is cancelled), but I can no longer re-deploy the stream anymore, as it fails complaining something like `Failed to upload the package. Package [xxxx] in Repository [local] already exists`.

Additionally, the Audit Log does not show the `UNDEPLOY` step, although the stream appears as `UNDEPLOYED`.

**Release versions:**
Data Flow - 2.2.1.RELEASE
Skipper - 2.1.2.RELEASE
Deployed into K8s using the official Helm Chart with almost default values.

**Steps to reproduce:**
Create simple stream. Deploy it. Delete release from Skipper. Try to re-deploy it again.

**Comment by ilayaperumalg** [comment]
Hi @emedina , Thanks for reporting this. Could you tell why you would have to delete the release from Skipper? The Skipper server isn't intended for changing its state directly when used along with SCDF

**Comment by emedina** [comment]
Good point. To be honest, I was just playing around as, I could do it, I thought the whole platform would sync up accordingly. If you tell me that, even when it's possible, it should not be done, then it's fine, but I have to prepare a runbook for my users in order to not allow them to do it.

See my concern?
